client/upload_client.go

```diff
@@ -0,0 +1,93 @@
+package client
+
+import (
+	"errors"
+	"io"
+	"os"
+
+	qs "github.com/yunify/qingstor-sdk-go/service"
+)
+
+type Uploader struct {
```

Comment:
Follow the style like https://github.com/yunify/qingstor-sdk-go/blob/master/client/image/image.go#L99 , Upload is enough